pkgs/top-level/ocaml-packages.nix

```
@@ -324,8 +324,14 @@ let
       inherit (pkgs) opam git mercurial coreutils gnutar bzip2;
     };
 
+    dune-rpc = callPackage ../development/ocaml-modules/dune-rpc { };
+
     dune-site = callPackage ../development/ocaml-modules/dune-site { };
 
+    fiber = callPackage ../development/ocaml-modules/fiber { };
+
+    xdg = callPackage ../development/ocaml-modules/xdg { };
```

```suggestion
    fiber = callPackage ../development/ocaml-modules/fiber { };

    xdg = callPackage ../development/ocaml-modules/xdg { };
```

`fiber` and `xdg` are not in alphabetical order.

---

I reordered the list alphabetically.